packages/asmfmt/package.yaml

```
@@ -0,0 +1,18 @@
+---
+name: asmfmt
+description: |
+  Go Assembler Formatter
+  This will format your assembler code in a similar way that gofmt formats your Go code.
+homepage: https://github.com/klauspost/asmfmt
+licenses:
+  - MIT
+languages:
+  - Assembly
+categories:
+  - Formatter
+
+source:
+  id: pkg:golang/github.com/klauspost/asmfmt/cmd/asmfmt@v1.3.2
```

For Renovate to successfully pick up new versions we need to separate the package name from the submodule (I forget the "Go" name for this).

```suggestion
  id: pkg:golang/github.com/klauspost/asmfmt@v1.3.2#cmd/asmfmt
```